docker-compose.yaml

```diff
@@ -22,9 +22,9 @@ services:
     image: datastrato/hive:2.7.3-no-yarn
     ports:
       - "3307:3306"
-      - "9000:9000"
-      - "9083:9083"
-      - "50070:50070"
+      - "9003:9000"
```

Comment:
use `19000` not `9003` for 9000 port to keep consistent with other port changes.

Reply:
Sorry I just merged this, @shaofengshi can you please submit another PR?

Reply:
here in the "hive:2.7.3-no-yarn", as the first port was forwarded from 3307 to 3306, so I choose the "+1" for all ports in this image; as 9001 port is already used in Mac (Port 9001 is used by the NMC server to make RPC calls between the NMC client (Java instance) and the NMC server. ), so changed to 9003, which is not used by any known services. So I think don't need to change to 19000, as it will be inconsitent with others in this docker container. @FANNG1

Reply:
It's a little complex to know the rules,  I prefer to keep a simple +10000 rule

Reply:
ok, I will raise another PR for it.